src/maps.ts

```diff
@@ -438,15 +438,12 @@ Spectrum flags are details to be aware of when reading the spectra.
   @param lower lower first 32 bytes
   @param higher higher 32 bytes
   @return Object with all the flags*/
-export function getWdfSpectrumFlags(
-  lower: number,
-  higher: number,
-): WdfSpectrumFlags {
+export function getWdfSpectrumFlags(flag: bigint): WdfSpectrumFlags {
   return {
-    saturated: (lower & 1) !== 0,
-    error: (lower & 0b10) !== 0,
-    cosmicRay: (lower & 0b100) !== 0,
-    errorCode: higher >>> 31,
+    saturated: !!(flag & BigInt(1)),
```

Comment:
```suggestion
    saturated: !!(flag & 1n),
```

why not use the bigint literal syntax?